gp-templates/translations.php

```diff
@@ -183,7 +183,12 @@
 		if ( 'rejected' == $status ) continue;
 ?>
 	<div class="box status-<?php echo $status; ?>"></div>
-	<div><?php echo $status; ?></div>
+	<div>
+		<?php if ('current' == $status ) { _e( 'Current', 'glotpress' ); } ?>
+		<?php if ('waiting' == $status ) { _e( 'Waiting', 'glotpress' ); } ?>
+		<?php if ('fuzzy' == $status ) { _e( 'Fuzzy', 'glotpress' ); } ?>
+		<?php if ('old' == $status ) { _e( 'Old', 'glotpress' ); } ?>
+	</div>
```

Comment:
The above replacement code doesn't quite do what the original code did, for example if the status isn't one of the ones listed, the old code would still output it, but the new code will not.

Also using 4 if's isn't very good as they all have to be evaluated, either going to a if/else structure or case switch statement would make more sense and let you fall through to the default of outputting the untranslated status.